routes/routes.php

```diff
@@ -86,12 +86,20 @@ function () {
                     'signature/store',
                     \Railroad\Railforums\Controllers\UserForumSignaturesController::class.'@store'
                 )
+                    ->middleware([
+                        \Railroad\Railforums\Middleware\HTMLSanitization::class,
+                        \Railroad\Railforums\Middleware\SignatureHTMLSyntaxCheck::class
+                    ])
                     ->name('railforums.signature.store');
 
                 Route::patch(
                     'signature/update/{id}',
```

Comment:
There's 3 similar routes for update signature, are these used?  Does the app also allow you do update the signature?
![image](https://github.com/railroadmedia/railforums/assets/108689774/afafa0ac-6c77-4cf7-8c53-97b34e8eca34)